src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -204,22 +204,21 @@ Copyright (c) .NET Foundation. All rights reserved.
     packages references.
     ============================================================
     -->
-  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectResolvedSDKReferencesDesignTime"
-             AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
-  
-  <Target Name="CollectSDKReferencesDesignTime"
-          Returns="@(_SDKReference)"
-          DependsOnTargets="ResolvePackageDependenciesDesignTime">
+  <UsingTask TaskName="Microsoft.NET.Build.Tasks.CollectSDKReferencesDesignTime"
+         AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
 
-    <CollectResolvedSDKReferencesDesignTime
-          ResolvedSdkReferences="@(SdkReference)"
-          DependenciesDesignTime="@(_DependenciesDesignTime)">
+  <Target Name="CollectSDKReferencesDesignTime"
+          Returns="@(_SDKReference)">
```

Comment:
@abpiskunov I think you are thinking of the ResolvePackageReferences target. `CollectPackageReferences` is a separate target that currently just returns all the PackageReference items from evaluation. It is an extensibility point for some other tools\package managers (say like Paket) to inject unresolved PackageReferences dynamically. It still requires a DT build (which is being performed anyway to run this target) but doesn't read the assets file and is quick.